cluster/up.sh

```diff
@@ -28,3 +28,5 @@ for node in $(./cluster/kubectl.sh get nodes --no-headers | awk '{print $1}'); d
     ./cluster/cli.sh ssh ${node} -- sudo systemctl restart openvswitch
     ./cluster/cli.sh ssh ${node} -- sudo systemctl restart NetworkManager
 done
+
+./cluster/kubectl.sh create -f ./cluster/cluster_reader.yaml
```

Comment:
This is part of the BearEach at tests

Reply:
The cluster-reader creation part of it must be part of the cluster-up before we cluster-sync,
else the reconcile won't create the cluster-role

beside that having a test-user doesn't depend on the `cluster-sync` and might be helpful for other uses as well
as it simulates OCP env that have cluster-reader
and doesn't need to recreated again and again imo